tests/highlevel/test_messages.py

```diff
@@ -1059,6 +1057,80 @@ def test_multiple_files(self):
                 '2\tpython-line\teggs.py\t17\tNone\tNone\tNone'),
         ])
 
+    def test_vs_django(self):
+        with self.launched(args={'options': 'DJANGO_DEBUG=True'}):
+            self.send_request(
```

Comment:
We will also need tests for remote debugging (attach request).

Reply:
Will do this as a part of attach: https://github.com/Microsoft/ptvsd/issues/248